package.json

```
@@ -8,28 +8,26 @@
 		"start": "npm run dev",
 		"dev": "vite dev",
 		"dev:cron": "wrangler dev --test-scheduled",
-
 		"generate:types": "wrangler types api/src/env.d.ts",
 		"db:setup": "wrangler d1 migrations apply vms-local --local",
 		"db:seed": "wrangler d1 execute vms-local --local --file=./db/seeds/seed-data.sql",
-
 		"build": "vite build",
 		"prepreview": "npm run build",
 		"preview": "vite preview",
-
 		"predeploy:dev": "npm run build",
 		"deploy:dev": "wrangler deploy --minify --env dev",
 		"predeploy:prod": "npm run build",
 		"deploy:prod": "wrangler deploy --minify --env production",
-
 		"format": "dprint fmt --staged",
 		"typecheck:api": "tsc -p ./api --noEmit",
 		"typecheck:client": "tsc -p ./src --noEmit",
 		"lint:js": "eslint",
 		"lint": "npm run typecheck:api && npm run typecheck:client && npm run lint:js",
-
-		"test": "vitest",
-		"coverage": "vitest run --coverage"
+		"test:unit": "vitest",
+		"coverage": "vitest run --coverage",
+		"test:api": "hurl --variables-file .dev.vars api/api_tests/*.hurl --verbose --test",
+		"test:api:ci": "hurl --glob 'tests/**/*.hurl' --continue-on-error --test --jobs 4 --max-time 300 --no-color --output-tap results.tap --variable-file .env.vars",
```

```suggestion
		"test:api": "hurl --glob \"./api/**/*.hurl\" --test --continue-on-error --jobs 4 --max-time 300 --output-tap \"./api-test-results.tap\"  --variables-file \"./.dev.vars\" --verbose",
		"test:api:ci": "hurl --glob \"./api/**/*.hurl\" --test --continue-on-error --jobs 4 --max-time 300 --output-tap \"./api-test-results.tap\" --no-color",
```

Let's keep the commands as similar as possible. Also, on CI, there is no `.dev.vars` file, it has to be provided from somewhere else.

---

Question: Is there a convention why it would be named result.tap?